docs/File-Formats.md

```diff
@@ -783,6 +783,7 @@ A fusion data file is a two dimensional matrix with one gene per row.  For each
 7. **RNA_support**: Fusion detected from RNA sequence data, "yes" or "no".
 8. **Method**: Fusion detected algorithm/tool.
 9. **Frame**: "in-frame" or "frameshift".
+10. **Fusion_Status (OPTIONAL)**: An assessment of the mutation type (i.e., "SOMATIC", "GERMLINE", "UNCALLED", etc.)
```

Comment:
good. (keeping docs current)

Reply:
Small comment: I'm not sure what you mean with 'etc'. Currently it's not clear if this should contain specific values or is free text. Perhaps it's best that this column has the same functionality as "Mutation_Status" in the MAF format and refer to that.


Mutation_Status in File-Formats:

26. **Mutation_Status (Optional):** "Somatic" or "Germline" are supported by the UI in Mutations tab. "None", "LOH" and "Wildtype" will not be loaded. Other values will be displayed as text.

Reply:
We should change  "in Mutations tab" to in Mutations tab and OncoPrint"